### DISCUSSION TITLE
Eu posso digger quelas tem mania de perguntar “ o que é isso”

### DISCUSSION THREAD

**Discussion by luis007007** [community/community#132460] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

E vc tem um a seguidora assim, a única coisa que eu tenho dela é os e-mail de recuperação que já foram trocados, mais o que ela fez não se faz, é de novo desculpe meu descontrole trabalho com duas máquinas mais celular e quatro e-mails, se talvez conseguisse localizar, eu não sou programador.

**Comment by luis007007** [comment]
Pelo e-mail não consegue?

**Comment by luis007007** [comment]
E o nome

**Reply by luis007007** [reply]
Luciana pinaffo

**Comment by luis007007** [comment]
Ela opera pelo iPhone, e chega pra mim tudo que ela deleta

**Comment by luis007007** [comment]
A ela também faz parte de uma associação

**Comment by luis007007** [comment]
Da para colocar foto aqui?

**Comment by luis007007** [comment]
![IMG_6746](https://github.com/user-attachments/assets/11efdb6c-854a-4d5f-8c1a-463b626506ca)

**Comment by LiteBrite82** [comment]
Hi @luis007007 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [Microsoft's Policy Page](https://www.microsoft.com/en-us/digitalsafety/policies). 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
